macros/modern.asm

```diff
@@ -0,0 +1,12 @@
+; Modern support for legacy pokecrystal and fanhacks that use older codebase
+; Allows porting scripts with as few edits as possible.
+
+; macros/rst.asm
+DEF farcall EQUS "callba"
+DEF callfar EQUS "callab"
```

Comment:
#759 specifically requested modern *music* script support for older projects. Throwing in extra undiscussed stuff isn't an improvement, it's a breakage: in this case, `farcall` has always been defined, and this would be a compile error.

Reply:
Also if older projects use older rgbds versions, they may not support `DEF` syntax. (I assume that's why you did `transpose: MACRO` and not `MACRO transpose` below, too.)

Reply:
older rgbds variants of this may require different syntax, yes. It might be best to give that a shot after this PR's ready to merge